apps/hash-ai-worker-py/app/typesystem/entity_type.py

```diff
@@ -31,3 +76,34 @@ class EntityTypeSchema(
     examples: list[dict[str, Any]] | None = None
     links: dict[str, Array[OneOf[EntityTypeReference] | EmptyDict]] | None = None
     all_of: list[EntityTypeReference] | None = Field(None, alias="allOf")
+
+    async def create_entity_type(
+        self,
+        *,
+        actor_id: UUID,
+    ) -> Annotated[Any, "PropertyTypeAnnotation"]:
```

Comment:
This type hint seems to be wrong

Reply:
[`aa90f2a` (#2708)](https://github.com/hashintel/hash/pull/2708/commits/aa90f2a2b59e5443ceef891bc8a272bb9277673e)